htdocs/adherents/card.php

```diff
@@ -416,7 +416,9 @@
 			}
 
 			if (!$error) {
-				$result = $object->update($user, 0, $nosyncuser, $nosyncuserpass);
+				$nosyncthirdparty = (!getDolGlobalString('MEMBER_NO_SYNC_LINKED_THIRDPARTY') ? 0 : 1);
```

Comment:
Wy not just 
`$nosyncthirdparty = getDolGlobalInt('MEMBER_NO_SYNC_LINKED_THIRDPARTY');`

Reply:
@eldy Yes, you're right, it's an old reflex 😄